Implement Alert component

thgreasi opened the pull request:
Normal (not in styleguide)
![screenshot-2017-12-5 storybook 1](https://user-images.githubusercontent.com/1295829/33597585-8b336f7a-d9a8-11e7-8d9b-8e536280bc5e.png)
Word wrapping
![screenshot-2017-12-5 storybook 2](https://user-images.githubusercontent.com/1295829/33597588-8da7da98-d9a8-11e7-9c86-114a15a6928c.png)
Emphasized (as found in styleguide)
![screenshot-2017-12-5 storybook 3](https://user-images.githubusercontent.com/1295829/33597598-9a778fca-d9a8-11e7-985f-883754ca9064.png)
Outlined
![screenshot-2017-12-5 storybook 4](https://user-images.githubusercontent.com/1295829/33597591-90a1bb60-d9a8-11e7-962a-2933fe6a3af1.png)
Plaintext (with whitespace padding to align sensisibly with other nearby controls)
![screenshot-2017-12-5 storybook 5](https://user-images.githubusercontent.com/1295829/33597592-92a32124-d9a8-11e7-81af-84b0881976f0.png)


Resolves: #203
Change-Type: minor

konmouz commented:
@thgreasi, I have some issues:
- Not sure we need 'Normal', 'Outlined' should be used when a message is not emphasized.
- 'Plaintext' doesn't need an 'x' button, these messages can't be removed
- The font kerning looks a bit weird. Make sure you use Roboto 16px with 0 kerning everywhere except 'Plaintext' that should be 14px

thgreasi commented:
* Changed that. 'Outlined' is the new 'Normal'!
* Removed the 'x' button from the 'Plaintext' story
* Added a `font-kerning: none;` just in case, even though, this was not present in zeplin and I would guess that it just might be Firefox rendering things slightly differently. Re-checked the font weights & families.

![screenshot from 2017-12-05 13-24-04](https://user-images.githubusercontent.com/1295829/33604904-4b722652-d9c0-11e7-9a88-60d2cecf0293.png)
